app/views/articles/show.html.erb

```diff
@@ -1,16 +1,49 @@
-<h1><%= @article.title %></h1>
-<p><%= @article.body %></p>
-<ul>
-  <li><%= link_to 'Edit', edit_article_path(@article) %></li>
-  <li>
-    <%= link_to 'Destroy',
+
```

Comment:
Please review the indentation of this file.

Reply:
Done!